docs/src/prototypes/MenuButton/MenuButton.tsx

```diff
@@ -0,0 +1,198 @@
+import {
+  Accessibility,
+  AccessibilityBehavior,
+  Button,
+  ButtonProps,
+  Menu,
+  MenuProps,
+  Ref,
+  ShorthandValue,
+} from '@stardust-ui/react'
+import * as _ from 'lodash'
+import * as keyboardKey from 'keyboard-key'
+import * as PopperJS from 'popper.js'
+import * as React from 'react'
+import { Manager, Reference, Popper } from 'react-popper'
+
+import { focusButton, focusMenuItem, focusNearest } from './focusUtils'
+import menuButtonBehavior from './menuButtonBehavior'
+import MenuButtonWrapper from './MenuButtonWrapper'
+
+export interface MenuButtonProps {
+  accessibility?: Accessibility
```

Comment:
maybe we should omit this prop from `MenuButton`'s API - because, essentially, we don't expect this behavior to be overriden, given that `MenuButton` component (which is not even a Stardust component) is introduced just for the sake of matching the exact behavior described here: https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html .

Reply:
Agree, removed 👍